packages/distributed/distributed.0.6.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "essdotteedot <essdotteedot@gmail.com>"
+authors: [ "essdotteedot <essdotteedot@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/essdotteedot/distributed"
+dev-repo: "git+https://github.com/essdotteedot/distributed.git"
+bug-reports: "https://github.com/essdotteedot/distributed/issues"
+doc: "https://essdotteedot.github.io/distributed/"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build & >= "1.11.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11.0"}
```
The problem with the revdeps comes from this. Dune should not be tagged as a build dependency (see #14266)

Reply:
Thanks. I've made the suggested changes.